### PR TITLE
chore: Delete MetaStore and RootStore

### PR DESCRIPTION
**Pull request by ivasilov**

### CODE REVIEW

File: apps/studio/components/interfaces/TableGridEditor/TableGridEditor.tsx [hunk 1]
```diff
@@ -188,11 +189,15 @@ const TableGridEditor = ({
   }
 
   const onSqlQuery = async (query: string) => {
-    const res = await meta.query(query)
```
Comment (joshenlim) [inline apps/studio/components/interfaces/TableGridEditor/TableGridEditor.tsx:191]: i'm actually not sure if onSqlQuery is being used actually - i can't find the trace inside SupabaseGrid which calls this method anywhere

File: apps/studio/components/interfaces/Database/Triggers/CreateTrigger.tsx [hunk 2]
```diff
@@ -237,7 +237,7 @@ interface CreateTriggerProps {
 
 const CreateTrigger = ({ trigger, visible, setVisible }: CreateTriggerProps) => {
   const { project } = useProjectContext()
-  const { ui, meta } = useStore()
+  const { ui } = useStore()
```
Comment (joshenlim) [inline apps/studio/components/interfaces/Database/Triggers/CreateTrigger.tsx:240]: i think ui is also not used?

File: apps/studio/components/interfaces/Auth/Policies/Policies.tsx [hunk 3]
```diff
@@ -38,6 +40,8 @@ const Policies = ({
   const { project } = useProjectContext()
   const snap = useTableEditorStateSnapshot()
 
+  const { ui } = useStore()
```
Comment (joshenlim) [inline apps/studio/components/interfaces/Auth/Policies/Policies.tsx:43]: these imports intentional?

File: apps/studio/components/interfaces/Docs/Description.tsx [hunk 4]
```diff
@@ -53,22 +56,26 @@ const Description = ({ content, metadata, onChange = noop }: DescrptionProps) =>
...
+        await timeout(500)
 
-      if (res.error) {
-        ui.setNotification({
-          error: res.error,
-          category: 'error',
-          message: `Failed to update description: ${res.error.message}`,
-        })
-      } else {
         ui.setNotification({
```
Comment (alaister) [inline apps/studio/components/interfaces/Docs/Description.tsx:69]: Worth converting to `toast.success`?

**Review by alaister — APPROVED** [review]
LGTM! One tiny nit:

File: apps/studio/components/interfaces/Docs/Description.tsx (hunk 4 above)
Comment (joshenlim) [reply apps/studio/components/interfaces/Docs/Description.tsx:69]: ^ i'll do all these in a separate PR!

**Pull request merged**